src/applications/vaos/services/mocks/v2/confirmed.json

```diff
@@ -148,7 +148,7 @@
         "start": "2024-10-13T15:00:00Z",
         "status": "booked",
         "created": "2024-11-01T00:00:00Z",
-        "cancellable": true,
+        "cancellable": false,
```

Comment:
This is a covid-19 mock appointment and the cancellable property is default to `false`.

Reply:
I was curious why this was changed for this PR though. It seems unrelated or was this part of testing something specific?

Reply:
@JunTaoLuo This PR was failing at an E2E stability test. Even after a rerun on the specific stability test, CI continues to show the same failure. It just needed a new CI build. While this json file is unrelated to removing font awesome icons, updating the mock file killed two bird with one stone...create a new CI and cleanup our mock json file.

Reply:
Ah I see, but let's not rely on this method. 

Can you file a test issue using this [template](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=ldelacosta&labels=testing%2C+appointments%2C+needs-grooming&projects=&template=vaos-test-issue.md&title=) and disable the test in this PR? If the test is flaky it will ruin someone else's day too so let's just disable it while it's being investigated. Add a link to the filed issue in a comment above the disabled test please. Thanks!

Reply:
I will definitely save the template in my favs. Ultimately, there was nothing wrong with the stability test. After filing an issue with Platform Support, they indicated the container failed to initialize. All tests passed after they rerun the container. Would you approve this PR again?

Reply:
Ah so it's flaky infrastructure. That's harder to fix than flaky tests 😄